doc: Add Tracker edition information (#27)
TheThingsIndustries/generic-node-docs: doc/content/tracker-edition/software/te-firmware/_index.md

---
title: "Tracker Edition Firmware"
description: ""
weight: -1
distributions: null
---

> The {{% gnte %}} [firmware](https://github.com/TheThingsIndustries/generic-node-te) allows for building multiple applications using [Semtech LR1110](https://www.semtech.com/products/wireless-rf/lora-edge/lr1110) modem.

Reviewer: ```suggestion
The {{% gnte %}} [firmware](https://github.com/TheThingsIndustries/generic-node-te) allows for building multiple applications using the [Semtech LR1110](https://www.semtech.com/products/wireless-rf/lora-edge/lr1110) modem.
```

> The STM32L073 host MCU communicates with the LR1110 modem over SPI bus, and it is able to perform multiple functionalities such as Wi-Fi and GNSS scanning and transmit the information using LoRaWAN.

Reviewer: ```suggestion
The STM32L073 host MCU communicates with the LR1110 modem over SPI bus, and it is able to perform functions such as Wi-Fi and GNSS scanning, and transmit the information using LoRaWAN.
```

> A simple Tracker example app joins the LoRaWAN network and performs a clock sync, and then periodically performs a Wi-Fi and GNSS scans and streams the scanned results to the network.

Reviewer: ```suggestion
A simple Tracker example app joins the LoRaWAN network and performs a clock sync, periodically performs Wi-Fi and GNSS scans, and streams the scanned results to the network.
```

> The application server solves the streamed data location coordinates provides an accurate location.

Reviewer: ```suggestion
The application server solves the streamed data location coordinates and provides an accurate location.
```
